app/Http/Controllers/FriendController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Http\Controllers;
+use DB;
+use Auth;
+use User;
+use Illuminate\Http\Request;
+
+class FriendController extends Controller
+{
+    public function Index(){
+        $user = \App\User::find(Auth::user() -> username);
```

Comment:
if you're looking for the current user, just Auth::user()->username will do.

if you want to search a list of users. Add `use App\User;` at the top then you can just do User::find